scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-2c.yaml

```diff
@@ -69,6 +69,7 @@ proposals:
 - barclamp: ceph
   attributes:
     disk_mode: first
+    client_network: admin
```

Comment:
As commented on the bug, this seems to be a side issue of the public/admin/bmc networks all falling under the 10.162.24.X network, probably due to the systems being physical machines with only one eth interface?

so the ceph-__mon__ gets listening on the lower/first ip to be resolved to the hostname?, which is the admin network, while the configuration ips it gets are from the public.

For example:

node1: 
ceph-mon listens on 10.162.24.50 -> admin network
ceph config points the other node at 10.162.24.161 -> public network

node2:
ceph-mon listens on 10.162.24.51 -> admin network
ceph config points the other node at 10.162.24.160 -> public network


Like this, it would be impossible for the nodes to talk to each other as they are trying to connect to an address in which the ceph-mon is not listening.


Now Im not sure if this is expected from the config of the physical machines or its a real bug on the ceph cookbook as Im not that experienced with ceph.

Its expected from ceph to be always listening and connecting on the public network? In that case a modification of the cookbook is needed.